packages/app/src/systems/Core/components/CoinInput.tsx

```diff
@@ -219,6 +48,7 @@ export const CoinInput = forwardRef<HTMLInputElement, CoinInputProps>(
           ) : (
             <NumberFormat
               {...props}
+              className="coinInput--input"
```

Comment:
Maybe accept className from the props also and mix?

Reply:
✅